src/App.jsx

```diff
@@ -1,40 +1,46 @@
-import React from 'react';
+import React, { useState } from 'react';
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 import './App.scss';
+import { Tabs } from './components/Tabs';
+
+interface Tab {
+  id: string,
+  title: string,
+  content: string,
+}
 
 export const tabs = [
   { id: 'tab-1', title: 'Tab 1', content: 'Some text 1' },
   { id: 'tab-2', title: 'Tab 2', content: 'Some text 2' },
   { id: 'tab-3', title: 'Tab 3', content: 'Some text 3' },
 ];
 
-export const App = () => (
-  <div className="section">
-    <h1 className="title">
-      Selected tab is Tab 1
-    </h1>
-
-    <div data-cy="TabsComponent">
-      <div className="tabs is-boxed">
-        <ul>
-          <li className="is-active" data-cy="Tab">
-            <a href="#tab-1" data-cy="TabLink">Tab 1</a>
-          </li>
+export const App: React.FC = () => {
+  const [selectedTab, setSelectedTab] = useState(tabs[0]);
+  const currentTab = (tab: Tab) => setSelectedTab(tab);
 
-          <li data-cy="Tab">
-            <a href="#tab-2" data-cy="TabLink">Tab 2</a>
-          </li>
+  return (
+    <div className="section">
+      <h1 className="title">
+        Selected tab is Tab
+        &nbsp;
+        {selectedTab.id.slice(-1)}
```

Comment:
please don't call a function from the interface because it will be executed every time when the state change, so make sense to move it to the variable

Reply:
fixed, thank you for the advice )